cmd/fs-v1-multipart-common_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"time"
 )
 
-// TestFSIsBucketExist - complete test of isBucketExist
```

Comment:
Do we still need this unit test given that the function is refactored to work uniformly with XL and FS backends.